src/Homework5/task1.py

```diff
@@ -0,0 +1,25 @@
+'''
+Оформите решение задач из прошлых домашних работ в функции.
+Напишите функцию runner.
+(все станет проще когда мы изучим модули, getattr и setattr)
+runner() – все фукнции вызываются по очереди
+runner(‘func_name’) – вызывается только функцию func_name.
+runner(‘func’, ‘func1’...) - вызывает все переданные функции
+'''
+
+import funck
+
+
+def runner(*args):
```

Comment:
ideally, it shouldn't depend on function names - we can import the module and take all function names from it.